packages/sdk/src/types/ConnectorTypes.ts

```diff
@@ -242,5 +242,20 @@ export type AuthCredentials = GrafxTokenAuthCredentials | RefreshedAuthCredenden
 
 export enum AuthRefreshTypeEnum {
     grafxToken = 'grafxToken',
-    user = 'user',
+    any = 'any',
 }
+
+/**
+ * @param connectorId connector id
+ * @param type type of auth renewal needed
+ * @param headerValue the value of the chili proxy authorization header. This
```

Comment:
Can we mention actual header name? So instead of `chili proxy authorization header`, just mention `X-GRAFX-UNATHORIZED header`